docs/source/_templates/theme_variables.jinja

```diff
@@ -1,17 +1,17 @@
 {%- set external_urls = {
-  'github': 'https://github.com/williamFalcon/pytorch-lightning',
-  'github_issues': 'https://github.com/williamFalcon/pytorch-lightning/issues',
-  'contributing': 'https://github.com/williamFalcon/pytorch-lightning/blob/master/CONTRIBUTING.md',
-  'docs': 'https://williamfalcon.github.io/pytorch-lightning',
+  'github': 'https://github.com/PyTorchLightning/pytorch-lightning',
+  'github_issues': 'https://github.com/PyTorchLightning/pytorch-lightning/issues',
+  'contributing': 'https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CONTRIBUTING.md',
+  'docs': 'https://PyTorchLightning.github.io/pytorch-lightning',
```

Comment:
@williamFalcon this path/URL and the following linking specific section in new documentation...
Do you run it on readthecocs.org right?